tests/conftest.py

```diff
@@ -109,3 +115,26 @@ def tmp_dir():
 @pytest.fixture(autouse=True)
 def force_venv(mocker):
     mocker.patch.object(build.env, '_should_use_virtualenv', lambda: False)
+
+
+def pytest_report_header() -> str:
+    interesting_packages = [
+        'build',
+        'colorama',
+        'filelock',
+        'packaging',
+        'pip',
+        'pyproject_hooks',
+        'setuptools',
+        'tomli',
+        'virtualenv',
+        'wheel',
+    ]
+    valid = []
+    for package in interesting_packages:
+        with contextlib.suppress(ModuleNotFoundError):
+            valid.append(f'{package}=={metadata.version(package)}')
+    reqs = ' '.join(valid)
+    pkg_line = f'installed packages of interest: {reqs}'
+
+    return '\n'.join([pkg_line])
```

Comment:
```suggestion
    return pkg_line
```

Reply:
This was originally designed to build up several new lines. Though even where I first wrote it, it's now down to this line only. This is certainly the only one that currently makes sense here. But that's why it was weirdly written. :)